default.nix

```diff
@@ -339,16 +350,27 @@ in rec {
         mv $NIX_BUILD_TOP/temp "$PWD/deps/${pname}"
         cd $PWD
 
-        if [ -d ${deps}/deps/${pname}/node_modules ]; then
-          cp -r ${deps}/deps/${pname}/node_modules "deps/${pname}/node_modules"
+        ${if useWorkspace then ''
```

Comment:
Preserve the existing structure entirely by moving the top-level `node_modules` into `deps/${pname}/`. if not using workspace structure, the top-level `node_modules` will be empty, except for a link to the main package in `deps/`.